runtime/src/lib.rs

```diff
@@ -380,7 +380,7 @@ pub const GLMR: Balance = 1_000_000_000_000_000_000;
 
 parameter_types! {
 	/// Moonbeam starts a new round every hour (600 * block_time)
-	pub const BlocksPerRound: u32 = 600;
+	pub const BlocksPerRound: u32 = 20;
```

Comment:
should be changed back before merged

Reply:
I can open a new PR that puts it in storage and allows root to update it. Should that be in this release?

Reply:
Just to summarize, Amar's PR #251 will not go into the v0.6 release, so this value should be reverted.